### ISSUE TITLE
Does not support Castle.Windsor v4.1

### ISSUE THREAD

**Issue by dahst** [adamconnelly/WindsorAppConfigFacility#3]
I'm working on a project that uses Castle.Windsor and WindsorAppConfigFacility. We upgraded Castle.Windsor to 4.1.0 recently but found that WindsorAppConfigFacility was still attempting to use 3.3.0 at runtime (even though the solution built without issues).

Since our usage of this library was fairly simple, we've swapped it out with some code to use Castle.Components.DictionaryAdapter (from Castle.Core) instead.

It might be worth updating the library, or setting the maximum supported version of Castle.Windsor dependency for NuGet.

**Comment by LiveDrunk** [comment]
Please upgrade your Castle package versions and publish the new version.
There are no code changes needed.

**Comment by adamconnelly** [comment]
Hi,

Sorry about this. For some reason I didn't seem to receive the email notification. I'll take a look at this as soon as possible.

**Comment by adamconnelly** [comment]
@dahst @LiveDrunk 

I've updated the library to support Castle 4.0.0+. I updated both packages to reference version 4.0.0 of Castle.Core and Castle.Windsor. The reason I did that is that from testing it looked like that would allow it to work with the latest stable versions of both libraries. Assembly versioning gets a bit confusing at times though, so let me know how you get on.

Thanks for the heads up - I was actually kinda surprised that anyone outside my work was using this!
